apps/ledger-live-mobile/tsconfig.json

```diff
@@ -13,7 +13,31 @@
     "rootDir": ".",
     "paths": {
       "~/*": ["src/*"],
```

Comment:
why not using these `~/` ? the `LLM@path` isn't very standard and we already have the tilde rule today so i'm not sure why introducing a third way of doing imports (we have local (the dot dot), absolute (the tilde), and now you introduce another absolute alternative)